config/application.rb

```diff
@@ -23,5 +23,6 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+    config.action_controller.include_all_helpers = false
```

Comment:
Why did you add this configuration?

Reply:
Normally, Rails loads all helper files in all views, which I don't prefer because it might lead to method names conflicting between different helpers. So, I changed a setting to stop this default behavior but i was wrong. Let me try another way with the addition of the helper inside the controller.